uni/UWaterloo.py

```diff
@@ -123,23 +122,24 @@ def scrapeCourseList(self, uw, term, subjectList):
         courseList = []
         # For each subject scrape courses
         for subject in subjectList:
-
             # Gets all courses based on term and subject
             for course in uw.term_subject_schedule(term, subject['subject']):
 
                 if course['catalog_number'] != prevClass or prevClass == '':
                     group = []
+                    # initialize course dictionary
+                courseDict = {'coursenum': course['catalog_number'], 'subject': subject['subject'],
```

Comment:
Put `coursenum` and `subject` on separate lines

Same for the subsequent attributes, makes it easier to read